app/models/user.rb

```diff
@@ -395,17 +395,9 @@ def onboarding
   delegate :show_onboarding_flow_on_login, to: :onboarding, allow_nil: true
 
   def vet360_contact_info
-    return nil unless VAProfile::Configuration::SETTINGS.contact_information.enabled &&
-                      (
-                        (!Flipper.enabled?(:remove_pciu, self) && vet360_id.present?) ||
-                        (Flipper.enabled?(:remove_pciu, self) && icn.present?)
-                      )
-
-    @vet360_contact_info ||= if Flipper.enabled?(:remove_pciu, self) && icn.present?
-                               VAProfileRedis::V2::ContactInformation.for_user(self)
-                             elsif !Flipper.enabled?(:remove_pciu, self) && vet360_id.present?
-                               VAProfileRedis::ContactInformation.for_user(self)
-                             end
+    return nil unless vet360_id.present? || icn.present?
+
+    @vet360_contact_info ||= VAProfileRedis::V2::ContactInformation.for_user(self)
   end
 
```

Comment:
Removed VAProfile::Configuration::SETTINGS.contact_information.enable and VAProfileRedis::ContactInformation